### ISSUE TITLE
error running bagtodepth 

### ISSUE THREAD

**Issue by ramanpreet9** [JaouadROS/bagtodepth#1]
when i run rosrun bag_to_depth bagtodepth newdepth
i changed newdepth to my bagfile path 
however i get the following error:  
[rospack] Error: package 'bag_to_depth' not found

does the bagfile (or something else) need to be in a particular folder?

edit:
After some debugging and looking at your code, i am able to read the bag file. However my bag file contains 3 topics. the one for depth is /camera/depth_registered/image_raw     which is probably different that yours.  I am getting degfault on image_msg->encoding because its probably nt reading teh correct material. 
Any suggestions on what changes i should make to read the depth correctly in my bag file?

**Comment by JaouadROS** [comment]
Did you solve the problem?
The images I'm extracting from the bag file are in 32FC1 type
And I was recording from this topic args="/asus1/depth/image  for depth images

**Comment by ramanpreet9** [comment]
no, but i think its because of some weird opencv 3 issue. As soon as i switched to opencv2 and ran my original program it worked fine to save the data. so, I didnt test your method on opencv2.
